tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -26,6 +31,34 @@ class TestUpgradeOCP(ManageTest):
     5. monitor cluster health
     """
 
+    def load_ocp_version_config_file(self, ocp_upgrade_version):
+        """
+        Loads config file to the ocs-ci config with upgrade version
+
+        Args:
+            ocp_upgrade_version: (str): version to be upgraded
+
+        """
+
+        short_ocp_upgrade_version = ".".join(ocp_upgrade_version.split(".")[0:2])
```

Comment:
What about 4.10 4.12 and so on.

Will this work here?

Reply:
fixed